src/shared/machine-pool.c

```diff
@@ -22,7 +22,8 @@ static int check_btrfs(void) {
         return F_TYPE_EQUAL(sfs.f_type, BTRFS_SUPER_MAGIC);
 }
 
-int setup_machine_directory(sd_bus_error *error) {
+int setup_machine_directory(sd_bus_error *error, bool setup_subvol, bool setup_quota)
+{
```

Comment:
Please do not insert line break between `)` and `{`.

I think it is better that this takes `Manager *` instead of two boolean.

Reply:
Oops, fixed.

re `Manager *`, this function is also invoked from systemd-machined without an importd manager.